app/src/main/java/com/github/libretube/adapters/CommentsAdapter.kt

```diff
@@ -16,7 +16,7 @@ class CommentsAdapter(private val comments: MutableList<Comment>):  RecyclerView
 
     fun updateItems(newItems: List<Comment>){
         comments.addAll(newItems)
-        notifyDataSetChanged()
+        notifyItemRangeInserted(comments.size, newItems.size)
```

Comment:
You should calculate `comments.size` before adding the `newItems` :)

Reply:
Hmm what's the difference?

Reply:
Added it but I don't know what saving it in a var should change xD